app/src/main/res/drawable/bg_black_gray_border.xml

```diff
@@ -1,12 +1,16 @@
 <?xml version="1.0" encoding="utf-8"?>
-<shape xmlns:android="http://schemas.android.com/apk/res/android"
-    android:shape="rectangle">
+<layer-list xmlns:android="http://schemas.android.com/apk/res/android">
```

Comment:
You're right- looks like it does fix the tests! I had erroneously stayed away from `ripple` because i saw it introduced a requirement of minSdkVersion 21 however placing these in the qualified drawable folder doesn't require we bump the minSdkVersion.